site/content/docs/commands/deploy.md

```diff
@@ -0,0 +1,22 @@
+#deploy command
```

Comment:
nit: Missing the space here, renders fine on the generated site but not if a user is looking directly in the docs folder on GitHub.

Reply:
done